### PR TITLE
feat: Move Security settings from Experimental Settings into Security Settings

### PR DESCRIPTION
**Pull request by segun**
## **Description**

We should move security alerts feature out of Settings > Experimental to Settings > Security & Privacy under the Security section below the show private key feature.

## **Related issues**

Fixes: [#1986](https://github.com/MetaMask/MetaMask-planning/issues/1986)

## **Manual testing steps**

1. Start Metamask
2. Open Settings > Experimental and see blockaid settings
3. Checkout this branch
4. Open Metamask
5. Open Settings > Security & Privacy and see blockaid settings under private key settings

### **Before**

https://github.com/MetaMask/metamask-mobile/assets/44811/0ed7131a-fcb4-4e57-b697-83496aa73d5b

### **After**

https://github.com/MetaMask/metamask-mobile/assets/44811/2e68562d-1674-41fe-9db9-3270d1bffb9b

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/Views/Settings/SecuritySettings/Sections/BlockaidSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        </Text>
+        <Text
+          color={TextColor.Alternative}
+          variant={TextVariant.BodyMD}
+          style={styles.desc}
+        >
+          {strings('experimental_settings.security_alerts_desc')}
+        </Text>
+      </View>
+      <View style={styles.switchElement}>
```
Comment (blackdevelopa) [inline app/components/Views/Settings/SecuritySettings/Sections/BlockaidSettings.tsx:70]: I will to update the styles.

Here's what `switchElement` should be https://github.com/MetaMask/metamask-mobile/blob/2a377e80923c6bcfe0a5fe102700ec2c73190f5f/app/components/Views/Settings/ExperimentalSettings/ExperimentalSettings.styles.ts#L28

File: app/components/Views/Settings/SecuritySettings/SecuritySettings.tsx [hunk 2]
```diff
@@ -978,6 +980,7 @@ const Settings: React.FC = () => {
           <RememberMeOptionSection />
         </View>
         <RevealPrivateKey />
+        {isBlockaidFeatureEnabled() && <BlockaidSettings />}
```
Comment (blackdevelopa) [inline app/components/Views/Settings/SecuritySettings/SecuritySettings.tsx:983]: Do we have a figma description of where the toggle should be in this screen?

**Comment by blackdevelopa** [comment]
Other than this, it looks good to me

File: app/components/Views/Settings/SecuritySettings/Sections/BlockaidSettings.tsx (hunk 1 above)
Comment (segun) [reply app/components/Views/Settings/SecuritySettings/Sections/BlockaidSettings.tsx:70]: haha, thanks, I was also wondering that it looks weird. but it seems like it looks same way on develop though.

File: app/components/Views/Settings/SecuritySettings/SecuritySettings.tsx (hunk 2 above)
Comment (segun) [reply app/components/Views/Settings/SecuritySettings/SecuritySettings.tsx:983]: No figma, but the issue said `after Reveal Passphrase`

**Review by jpuri — DISMISSED** [review]
LGTM

**Review by jpuri — DISMISSED** [review]

**Review by blackdevelopa — DISMISSED** [review]

File: app/components/Views/Settings/SecuritySettings/Sections/BlockaidSettings.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  const securityAlertsEnabled = useSelector(selectIsSecurityAlertsEnabled);
+  const navigation = useNavigation();
+
+  const toggleSecurityAlertsEnabled = () => {
+    const { PreferencesController } = Engine.context;
+
+    if (securityAlertsEnabled) {
+      PreferencesController?.setSecurityAlertsEnabled(false);
+      AnalyticsV2.trackEvent(
+        MetaMetricsEvents.SETTINGS_EXPERIMENTAL_SECURITY_ALERTS_ENABLED,
```
Comment (digiwand) [inline app/components/Views/Settings/SecuritySettings/Sections/BlockaidSettings.tsx:33]: I think we should update the variable names to exclude experimental now that we're moving the settings from experimental to security

File: app/components/Views/Settings/SecuritySettings/Sections/BlockaidSettings.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+          color={TextColor.Default}
+          variant={TextVariant.HeadingLG}
+          style={styles.heading}
+        >
+          {strings('app_settings.security_heading')}
+        </Text>
+      )}
+      <View style={styles.setting}>
+        <Text color={TextColor.Default} variant={TextVariant.BodyLGMedium}>
+          {strings('experimental_settings.security_alerts')}
```
Comment (digiwand) [inline app/components/Views/Settings/SecuritySettings/Sections/BlockaidSettings.tsx:60]: we should move the translations away from experimental_settings with the changes

**Review by digiwand — CHANGES_REQUESTED** [review]
for consistency, I think we should update the toggle button to be right-aligned with content on the left 

**current:** 
Screenshot 2024-02-20 at 10 28 31 AM

**example of other existing toggle settings:**
Screenshot 2024-02-20 at 10 28 02 AM
Screenshot 2024-02-20 at 10 28 15 AM

File: locales/languages/en.json [hunk 5]
```diff
@@ -892,7 +892,11 @@
     "network_provider": "Network provider",
     "token_nft_ens_subheading": "Token, NFT, and ENS autodetection",
     "security_check_subheading": "Security checks",
-    "symbol_required": "Symbol is required."
+    "symbol_required": "Symbol is required.",
```
Comment (blackdevelopa) [inline locales/languages/en.json:895]: Moved from `experimental settings` to `app settings`

**Review by blackdevelopa — APPROVED** [review]

File: app/core/Analytics/MetaMetrics.events.ts [hunk 6]
```diff
@@ -800,9 +800,7 @@ const events = {
   ),
 
   // Experimental Settings
```
Comment (digiwand) [inline app/core/Analytics/MetaMetrics.events.ts:802]:
```suggestion
  // Security Settings
```

**Review by digiwand — APPROVED** [review]

**Pull request merged**